adhocteam/Head-Start-TTADP
# [no ticket] add next steps to approved activity report view

## Description of change

The approved activity report should show specialist and grantee next steps.

## How to test
Visit an approved report page and check that the next steps table appears, near the bottom.

## Issue(s)



## Checklists

### Every PR


- [n/a] Meets issue criteria
- [n/a] JIRA ticket status updated
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
